README.md

```diff
@@ -544,25 +541,10 @@ Services to securely store your Docker images.
 
 # Useful Resources
 
-* [Blog](http://crosbymichael.com/) of [@crosbymichael][crosbymichael]
-* [Blog](https://gliderlabs.com/devlog/) of [@gliderlabs][gliderlabs]
-* [Blog](http://jasonwilder.com/) of [@jwilder][jwilder]
-* [Blog](http://jpetazzo.github.io/) of [@jpetazzo][jpetazzo]
-* [Blog](http://progrium.com/blog/) of [@progrium][progrium]
-* [Blog](http://sebgoa.blogspot.be/) of [@sebgoa][sebgoa]
-* [Blog](https://blog.codeship.com/) of [@codeship](https://github.com/codeship)
-* [Blog](https://blog.jessfraz.com/) of [@jessfraz][jessfraz]
-* [Container solutions](http://container-solutions.com/blog/)
-* [Container42](http://container42.com/)
+* __[Valuable Docker Links](http://www.nkode.io/2014/08/24/valuable-docker-links.html)__ High quality articles about docker! __MUST SEE__ 
+* Blogs [by @nickjanetakis](https://nickjanetakis.com/blog/tag/docker) [@crosbymichael](http://crosbymichael.com/) [@gliderlabs](https://gliderlabs.com/devlog/) [@jwilder](http://jasonwilder.com/) [@jpetazzo](http://jpetazzo.github.io/) [@progrium](http://progrium.com/blog/) [@sebgoa](http://sebgoa.blogspot.be/) [@codeship](https://blog.codeship.com/) [@jessfraz](https://blog.jessfraz.com/) * [@Container42](http://container42.com/) [Container solutions](http://container-solutions.com/blog/)
```

Comment:
should be listed like so:
* [@nickjanetakis](https://nickjanetakis.com/blog/tag/docker) 
* [@crosbymichael](http://crosbymichael.com/) 
* [@gliderlabs](https://gliderlabs.com/devlog/) 
* [@jwilder](http://jasonwilder.com/) 
* [@jpetazzo](http://jpetazzo.github.io/) 
* [@progrium](http://progrium.com/blog/) 
* [@sebgoa](http://sebgoa.blogspot.be/) 
* [@codeship](https://blog.codeship.com/) 
* [@jessfraz](https://blog.jessfraz.com/) 
* [@Container42](http://container42.com/) 
* [Container solutions](http://container-solutions.com/blog/)